common.lic

```diff
@@ -282,4 +282,27 @@ module DRC
       .map { |_name, properties| properties['abbrev'] }
       .each { |abbrev| fput("release #{abbrev}") }
   end
+
+  def necro_harvest_check
+    if !@necro_store
+      echo "Store material: false, dropping harvested material" if $debug_mode_ct
+      bput('drop material', 'you discard it')
+    else
+      echo "Store material: true, checking count of stored material" if $debug_mode_ct
+      materials = rummage('C material', than_container)
+      if materials.scan(/(?=a \S+ \S+ of \S+ material)/).count >= @necro_count
+        bput('drop material', 'you discard it')
+        echo "Already full on stored material, dropping harvested material." if $debug_mode_ct
+      else
+        quality = fput('glance left')
+        if quality['great'] or quality['excellent'] or quality['perfect']
```

Comment:
Our convention has been to use `unless @foo` in place of `if !@foo` as well as `||` instead of `or`.